web/blueprints/finance/forms.py

```diff
@@ -116,6 +116,11 @@ class BankAccountActivitiesImportForm(Form):
     do_import = BooleanField(u"Import durchführen", default=False)
 
 
+class BankAccountActivitiesImportManualForm(Form):
+    account = SelectField(u"Bankkonto", coerce=int)
```

Comment:
Use a `QuerySelectField` here instead.
You can find several examples in `web/blueprints/user/forms.py` and the documentation at http://wtforms.simplecodes.com/docs/0.6/ext.html#module-wtforms.ext.sqlalchemy.fields.

This loads the choices asynchronously and integrates better with SQLAlchemy.
`form.account.data` will return a BankAccount directly.
The `get_label` argument can be a callable to display the correct values in the select field.
`form.account.choices = ...` is not required anymore.